.github/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+Pull Request Checklist
+ - [ ] Pull request includes a description of the change and the reason behind it.
+ - [ ] Pull request [uses keywords to close relevant issues](https://help.github.com/en/articles/closing-issues-using-keywords).
+ - [ ] Pull request includes unit tests for any new functionality.
+ - [ ] `CHANGELOG.md` has been updated.
+ - [ ] `_version.py` has been updated.
+ - [ ] README and docs have been updated (if applicable).
+ - [ ] `pytest` passes locally.
+ - [ ] `black autofocus tests --check` passes locally.
```

Comment:
TIL [black](https://github.com/python/black)! cool! @jsal13 have you seen this?

Reply:
Yeah, I'm a big fan. It saves me tons of time fiddling with line breaks.